notices/ltc-asset-transfer.vbs

```diff
@@ -111,55 +127,52 @@ Call start_a_new_spec_memo(memo_opened, True, forms_to_arep, forms_to_swkr, send
-    If send_to_other = "Y" then notice_recip = notice_recip & "Other, "
+'creating a list of who the memo was sent to for the case note
+notice_recip = "Resident, "
+If forms_to_arep = "Y" then notice_recip = notice_recip & "AREP, "
+If forms_to_swkr = "Y" then notice_recip = notice_recip & "SWKR, "
+If send_to_other = "Y" then notice_recip = notice_recip & "Other, "
 
-    notice_recip = trim(notice_recip)  'trims excess spaces of notice_recip
-    If right(notice_recip, 1) = "," THEN notice_recip = left(notice_recip, len(notice_recip) - 1)
+notice_recip = trim(notice_recip)  'trims excess spaces of notice_recip
+If right(notice_recip, 1) = "," THEN notice_recip = left(notice_recip, len(notice_recip) - 1)
 
-    Call write_bullet_and_variable_in_CASE_NOTE("MEMO sent to", notice_recip)
-    Call write_variable_in_CASE_NOTE("Content of the MEMO:")
-    Call write_variable_in_CASE_NOTE("The ownership of" & client & "'s assets must be transferred to" & spouse & " to avoid having them counted in future eligibility determinations. You are encouraged to do this as soon as possible. This transfer of assets must be done before" & client & "'s first annual renewal for " & renewal_date & ". Verification of the transfer can be provided at any time.")
-    Call write_variable_in_CASE_NOTE("At the first annual renewal in " & renewal_date & ", the value of all assets that list" & client & " as an owner or co-owner will be applied towards the Medical Assistance Asset limit of $3,000.00. If the total value of all countable assets for" & client & " is more than $3,000.00, Medical Assistance may be closed for " & renewal_date & ".")
-    Call write_variable_in_CASE_NOTE("---")
-    Call write_variable_in_CASE_NOTE(worker_signature)
-End if
+Call write_bullet_and_variable_in_CASE_NOTE("MEMO sent to", notice_recip)
+Call write_variable_in_CASE_NOTE("Content of the MEMO:")
+Call write_variable_in_CASE_NOTE("The ownership of" & client & "'s assets must be transferred to" & spouse & " to avoid having them counted in future eligibility determinations. You are encouraged to do this as soon as possible. This transfer of assets must be done before" & client & "'s first annual renewal for " & renewal_date & ". Verification of the transfer can be provided at any time.")
+Call write_variable_in_CASE_NOTE("At the first annual renewal in " & renewal_date & ", the value of all assets that list" & client & " as an owner or co-owner will be applied towards the Medical Assistance Asset limit of $3,000.00. If the total value of all countable assets for" & client & " is more than $3,000.00, Medical Assistance may be closed for " & renewal_date & ".")
+Call write_variable_in_CASE_NOTE("---")
+Call write_variable_in_CASE_NOTE(worker_signature)
 
-script_end_procedure_with_error_report("Success! Please review your MEMO and/or CASE:NOTE for accuracy.")
+script_end_procedure_with_error_report("**Review your MEMO and/or CASE:NOTE for accuracy.**")
```

Comment:
There's a newer version of the CLOSING ISSUE DOCUMENTATION. There are a couple of newer elements in there to include.